hypothesis-python/src/hypothesis/extra/lark.py

```diff
@@ -78,9 +79,15 @@ def __init__(self, grammar, start=None):
         check_type(lark.lark.Lark, grammar, "grammar")
         if start is None:
             start = grammar.options.start
+        if not isinstance(start, list):
+            start = [start]
         self.grammar = grammar
 
-        terminals, rules, ignore_names = grammar.grammar.compile()
+        if "start" in getfullargspec(grammar.grammar.compile).args:
```

Comment:
I feel like this would be better as a top level constant rather than calling getfillargspec each time

Reply:
Sure, can do.

Reply:
It's pretty cheap to do that though, as well as clear what's happening. Other options are much more fragile and require a bunch of extra imports.